rules/windows/registry/registry_set/registry_set_wdigest_enable_uselogoncredential.yml

```
@@ -5,6 +5,7 @@ description: Detects potential malicious modification of the property value of U
 references:
     - https://threathunterplaybook.com/hunts/windows/190510-RegModWDigestDowngrade/notebook.html
     - https://support.microsoft.com/en-us/topic/microsoft-security-advisory-update-to-improve-credentials-protection-and-management-may-13-2014-93434251-04ac-b7f3-52aa-9f951c14b649
+    - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1112/T1112.md#atomic-test-3---modify-registry-to-store-logon-credentials
```

```suggestion
    - https://github.com/redcanaryco/atomic-red-team/blob/73fcfa1d4863f6a4e17f90e54401de6e30a312bb/atomics/T1112/T1112.md#atomic-test-3---modify-registry-to-store-logon-credentials 
```
HI, we use permanent link when possible